getting-started/other-resources/trac.rst

```diff
@@ -4,10 +4,12 @@
 TRAC review tool
 ================
 
-Drupal TRAC review tool
------------------------
+The Trustworthy Repositories Audit & Certification (TRAC) review tool is a Drupal
+implementation of the TRAC checklist, an auditing tool to assess the
+reliability, commitment and readiness of institutions to assume long-term
+preservation responsibilities.
 
-Developed by MIT in a project led by Nancy McGovern, Head of
+The review tool was developed by MIT in a project led by Nancy McGovern, Head of
```

Comment:
Hi @sallain there are two lines in this paragraph going over 80 characters. If we can correct those that will be good.